kadai1/oamam/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"gopherdojo-studyroom/kadai1/oamam/imgconv"
+	"log"
+)
```

Comment:
グルーピングした方がみやすいかなと思います！(VScodeお使いでしたら、goimports入れたら勝手にグルーピングされます！)
```suggestion
import (
	"flag"
	"fmt"
	"log"

	"gopherdojo-studyroom/kadai1/oamam/imgconv"
)
```
https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Goland を使っていますので、golangci-lint と golint で自動修正かかるように設定しました！